Sources/SwiftyMarkdown/SwiftyMarkdown.swift

```diff
@@ -249,6 +249,9 @@ If that is not set, then the system default will be used.
 	public var bullet : String = "・"
 	
 	public var underlineLinks : Bool = false
+    
+    /// **iOS only** : Setting this boolean to true will force using the exact size set for each style if available.
+    public var ignoreDynamicFontSize : Bool = false
```

Comment:
This works and covers an genuine use case, but a better solution might be to use `scaledFont(for:, maximumPointSize:)` from `UIFontMetrics` on `SwiftyMarkdown+iOS.swift` instead?